packages/js/client/src/__tests__/core/wasm-wrapper.spec.ts

```diff
@@ -213,135 +211,101 @@ describe("wasm-wrapper", () => {
               schema: "",
               implements: [],
             },
-          }
-        }
-      ]
-    })
+          },
+        },
+      ],
+    });
 
     await expect(() =>
       pluginClient.getFile("ens/mock-plugin.eth", {
-          path: "./index.js",
-        })
+        path: "./index.js",
+      })
     ).rejects.toThrow("client.getFile(...) is not implemented for Plugins.");
   });
 
   test("subscribe", async () => {
-    const client = await getClientWithEnsAndIpfs();
-
-    const deploy = await client.invoke<string>({
-      uri: simpleStorageWrapperUri.uri,
-      method: "deployContract",
-      args: {
-        connection: {
-          networkNameOrChainId: "testnet",
+    const client = await getClient({
+      plugins: [
+        {
+          uri: memoryStoragePluginUri,
+          plugin: makeMemoryStoragePlugin({}),
         },
-      },
+      ],
     });
 
-    expect(deploy.error).toBeFalsy();
-    expect(deploy.data).toBeTruthy();
-    expect(deploy.data?.indexOf("0x")).toBeGreaterThan(-1);
-
-    const address = deploy.data;
-
     // test subscription
+    let expectedResults: number[] = [];
     let results: number[] = [];
     let value = 0;
 
     const setter = setInterval(async () => {
+      expectedResults.push(value);
+
       await client.invoke({
-        uri: simpleStorageWrapperUri.uri,
+        uri: simpleMemoryWrapperUri.uri,
         method: "setData",
         args: {
-          address,
           value: value++,
-          connection: {
-            networkNameOrChainId: "testnet",
-          },
         },
       });
     }, 4000);
 
     const getSubscription: Subscription<number> = client.subscribe<number>({
-      uri: simpleStorageWrapperUri,
+      uri: simpleMemoryWrapperUri.uri,
       method: "getData",
-      args: {
-        address: address,
-        connection: {
-          networkNameOrChainId: "testnet",
-        },
-      },
       frequency: { ms: 4500 },
```

Comment:
Can we use a shorter frequency, with 4.5s it takes 13.5s for this test to run, it's a bit much to spend on a single test

Reply:
We can probably get away with going 500ms/550ms for the generator/consumer.